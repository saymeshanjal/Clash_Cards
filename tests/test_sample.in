2
10 317 1
16 358 3
