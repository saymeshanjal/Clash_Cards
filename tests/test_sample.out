126 126 0
6307 6563 0
